### ISSUE TITLE
Double / in config multisite

### ISSUE THREAD

**Issue by thelittlefireman** [bunkerity/bunkerized-nginx#49]
hi,
When i use multisite, nginx modsecurity config path contains a double / : 

```bash
/etc/nginx/nextcloud.mydomain # cat modsecurity.conf
modsecurity on;
modsecurity_rules_file /etc/nginx/nextcloud.mydomain//modsecurity-rules.conf;
/etc/nginx/nextcloud.mydomain # cat ../syno.mydomain/modsecurity.conf 
modsecurity on;
modsecurity_rules_file /etc/nginx/syno.mydomain//modsecurity-rules.conf;
/etc/nginx/nextcloud.mydomain # cat ../syno.mydomain/modsecurity-rules.conf 
# process rules with disruptive actions

```

Thanks

**Comment by thelittlefireman** [comment]
i don't know if it's related but i'm faced some issue : 

```
time="2020-12-09T21:59:17+01:00" level=info msg="Wrote 100 bans from api to database."
[*] Multi site - syno.mydomain configuration done
[*] Multi site - nextcloud.mydomain configuration done
[*] Multi site - bitwarden.mydomain configuration done
[*] Multi site - portainer.mydomain configuration done
nginx: [emerg] "modsecurity_rules_file" directive Rule id: 9003138 is duplicated
 in /etc/nginx/nextcloud.mydomain/modsecurity.conf:2
[*] Running nginx ...
nginx: [emerg] "modsecurity_rules_file" directive Rule id: 9003138 is duplicated
 in /etc/nginx/nextcloud.mydomain/modsecurity.conf:2
[*] Running fail2ban ...
[*] Running crowdsec ...
==> /var/log/access.log <==

```

**Comment by bunkerity** [comment]
Hello @thelittlefireman,

Can you share your docker-compose.yml and any others configurations so I can reproduce and fix this issue please ? (don't forget to edit private info)

**Comment by thelittlefireman** [comment]
hi
@bunkerity do you have an email address/ private chat room when I can send you the configuration?

Thanks

**Comment by thelittlefireman** [comment]
@bunkerity for information i've test the dev docker tag and it's working perfectly :)

**Comment by bunkerity** [comment]
Fixed in v1.2.2 ! Thanks for your work @thelittlefireman.

Please note that I've opened a matrix chat [#bunkerized-nginx:matrix.org](https://matrix.to/#/#bunkerized-nginx:matrix.org) if you wish to discuss about the project.
